src/nwbinspector/nwbinspector.py

```diff
@@ -399,6 +399,7 @@ def inspect_all(
             nwbfiles = [in_path]
         else:
             raise ValueError(f"{in_path} should be a directory or an NWB file.")
+    nwbfiles = [nwbfile for nwbfile in nwbfiles if not nwbfile.name.startswith("._")] # skip macos sidecar files
```

Comment:
What would you think about moving this to just after line https://github.com/NeurodataWithoutBorders/nwbinspector/pull/470/files#diff-a2903eeebf6003fffce4fe2c3e4b6887381daaff8460e62ee6bad47894984b3aL397?

That way if someone did, for whatever reason, point the CLI at a single file path with that prefix, it wouldn't encounter a weird issue related to an empty iterable